spring-boot-project/spring-boot-docs/src/docs/antora/modules/reference/pages/data/sql.adoc

```diff
@@ -165,7 +165,7 @@ spring:
 [[data.sql.jdbc-template]]
 == Using JdbcTemplate
 
-Spring's `JdbcTemplate` and `NamedParameterJdbcTemplate` classes are auto-configured, and you can `@Autowire` them directly into your own beans, as shown in the following example:
```

Comment:
This is intentional. While the annotation is called `@Autowired`, we made a verb out of it, `@Autowire`.

Team: I think that's somewhat confusion. Maybe it's besser to change this to "and you can autowire them ..." ?